client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -136,16 +136,18 @@ export class CartItem extends React.Component {
 
 	render() {
 		let name = this.getProductName();
```

Comment:
I am shocked you did not destructure the props a bit: `const { cartItem, translate } = this.props;` 😱 (I know it was like that before... but that has not stopped you before ;P)